### ISSUE TITLE
AWS SSO permission set support

### ISSUE THREAD

**Issue by benkehoe** [awslabs/aws-cloudformation-iam-policy-validator#4]
AWS SSO permission sets are another place inline principal policies can be defined. Can support be added for checking them as well?

**Comment by mluttrell** [comment]
Yes.  The plan is to add support for permission sets and continue to expand resource policy coverage.
